compiler/rustc_lint/src/hidden_unicode_codepoints.rs

```diff
@@ -0,0 +1,161 @@
+use crate::{EarlyContext, EarlyLintPass, LintContext};
```

Comment:
Github says this file contains bidir control characters. This will break once bootstrap gets this lint, right?

Reply:
Would `#[doc = "...\u{...}..."] work to avoid directly writing the bidir control characters?

Reply:
Uh, that's a *really* good point, cc @estebank. Good catch!

For the stable and beta patches I'd keep it this way, mostly because this is the patch we shared under embargo with Linux distributions ahead of time and I don't want us to apply a different patch. I'll update the master PR as soon as possible.

Reply:
Ok this is a bit harder than I thought since the lint docs generator tool doesn't understand `#[doc = ""]`. I'll merge the PR to master without that change as I want nightly to be fixed ASAP, we'll take care of this in a future PR.